package.json

```diff
@@ -60,8 +60,9 @@
 		"prestart": "npm run -s install-if-deps-outdated",
 		"start": "cross-env WC_ADMIN_PHASE=development npm run build:packages && cross-env WC_ADMIN_PHASE=development npm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"node ./bin/packages/watch.js\"",
 		"pretest": "npm run -s install-if-no-packages",
-		"test:e2e": "npm explore @woocommerce/e2e-environment -- npm run test:e2e",
-		"test:e2e-dev": "npm explore @woocommerce/e2e-environment -- npm run test:e2e-dev",
+		"test:e2e": "bash ./bin/wait-for-build.sh && ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js",
+		"test:e2e-dev": "bash ./bin/wait-for-build.sh && DEBUG=1 ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js --dev",
+		"test:e2e-debug": "bash ./bin/wait-for-build.sh && DEBUG=1 ./node_modules/@woocommerce/e2e-environment/bin/e2e-test-integration.js --dev --debug",
```

Comment:
Instead of `npm explore` I am calling the `e2e-test-integration.js` directly, this allows `jest` to use our `babel.config.js`.
When using `npm explore` it won't use our `babel.config.js`.